src/js/common/schemaform/validation.js

```diff
@@ -115,7 +115,7 @@ export function uiSchemaValidate(errors, uiSchema, formData, formContext, path =
     }
 
     const validations = uiSchema['ui:validations'];
-    if (validations) {
+    if (validations && currentData) {
```

Comment:
We don't need to run our validation if there's no data, since that will be caught by the required validation.